app/models/Search.java

```diff
@@ -263,14 +263,18 @@ public boolean doingScrollScanNow() {
 		return doingScrollScanNow;
 	}
 
-	private void bulk(final Request request, final Serialization serialization) {
+	private void bulk(final Request request, final Serialization serialization,
+			final String scroll) {
 		boolean JSON_LD = serialization.equals(Serialization.JSON_LD);
+		boolean RDF_XML = serialization.equals(Serialization.RDF_XML);
 		try {
 			long lastTime = Calendar.getInstance().getTimeInMillis();
-			SearchResponse searchResponse = startInitialResponse();
+			SearchResponse searchResponse = startInitialResponse(scroll);
 			final SearchHits hits = getTotalHits(searchResponse);
 			if (JSON_LD)
 				getMessageOut().write("[");
+			if (RDF_XML)
+				getMessageOut().write("<root>");
```

Comment:
Does this yield valid RDF-XML? The root element looks very different from our usual RDF-XML requests (which contain namespace declarations in a root element called `rdf:RDF`), see:

`curl -L --header "Accept: application/rdf+xml" "http://lobid.org/resource?q=ehrenfeld"`

Also, it seems even with `scroll`, in our current deployment on production, we get a root element, see:

`curl -L --header "Accept: application/rdf+xml" "http://lobid.org/resource?q=ehrenfeld&scroll=true"`

You committed this to your branch a few days before I fixed https://github.com/lobid/lodmill/issues/463 so maybe this is no longer needed?